From d58fb951ccfc366c92d8bbbfdce47ad2f0141ef0 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 16 Feb 2018 15:21:43 -0800
Subject: [PATCH] TRITON-139 Triton's config-agents should poll at 120s
 interval as a stopgap Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
 Approved by: Marsell Kukuljevic <marsell@joyent.com>

---
 deps/sdc-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index d3fbcc5..bda632a 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit d3fbcc591c2f57f202b699e7cd5d8e70b9c1efc4
+Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
-- 
2.21.0

